flow/embedded_views.h

```diff
@@ -183,6 +183,10 @@ class ExternalViewEmbedder {
  public:
   ExternalViewEmbedder() = default;
 
+  // This will return true after pre-roll if any of the embedded views
```

Comment:
Doesn't have to be in this PR, but we will need a way to clear `active_composition_order_` if we're redoing the frame, maybe by adding something like `CancelFrame` here?

Reply:
Yup, will do it as a follow-up PR.